src/daemon/keymap.c

```diff
@@ -968,27 +1088,27 @@ void handle_legacy_6kro_input(unsigned char* kbinput, const unsigned char* urbin
 void hid_kb_translate(unsigned char* kbinput, int length, const unsigned char* urbinput, int legacy){
     if(legacy) {
         switch(length) {
-            case 2: // K65 Media keys
+            case NKRO_K65_MEDIA_IN: // K65 Media keys
```

Comment:
These macros should be named LEN somewhere and not be bundled with the _IN macros. The _IN ones are not lengths, but the first byte value. But also, why are these modified in the first place?